connector/src/main/java/com/linker/connector/express/PostOffice.java

```diff
@@ -30,6 +32,9 @@
     @Autowired
     ExpressDeliveryFactory expressDeliveryFactory;
 
+    @Autowired
```

Comment:
Better to use `@RequiredArgsConstructor` and declare these dependencies as `final`

Reply:
Yes, I will create a new MR to fix the dependencies and the typo